build.gradle

```diff
@@ -753,10 +753,15 @@ subprojects {
 
     // TODO: it's a bug that annotatedlib:guava requires the error_prone_annotations dependency.
     annotatedGuava "com.google.errorprone:error_prone_annotations:${versions.errorprone}"
-    annotatedGuava ('org.checkerframework.annotatedlib:guava:30.1.1-jre') {
+    annotatedGuava ('org.checkerframework.annotatedlib:guava:33.1.0-jre') {
       // So long as Guava only uses annotations from checker-qual, excluding it should not cause problems.
       exclude group: 'org.checkerframework'
     }
+    // Without this, running the Checker Framework leads to "class file for
+    // javax.annotation.meta.TypeQualifierDefault not found".  I was not able to
+    // figure out how to use Gradle's exclusions (of transitive dependencies) in
+    // order to avoid the need for this.
+    annotatedGuava 'com.google.code.findbugs:jsr305:3.0.2'
```

Comment:
This line is needed because the pom file in Maven central doesn't have Guava's dependencies. See: https://central.sonatype.com/artifact/org.checkerframework.annotatedlib/guava/dependencies

Compared to Google's version:
https://central.sonatype.com/artifact/com.google.guava/guava/dependencies

You'll need to release annotated Guava to fix this. (I'm also ok if you want to merge this work around.)